index.html

```diff
@@ -93,7 +93,7 @@ <h5>Tonic</h5>
   </section>
   
   <section class="multi">
-    <img class="imagen-contenedor" src="assets/multi post.png" alt="">
```

Comment:
I have a couple of optional suggestions for you to enhance the overall professional look and adherence to the provided [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design) for your portfolio project:

- [ ] **[OPTIONAL]** It would be great if you could align the `background image` of your hero section. Currently, it **appears a bit cropped and doesn't match the image in the Figma template.** 🙋‍♂️

- [ ] **[OPTIONAL]** Please double-check the **position of your navbar** and main text to ensure they are accurately placed according to the Figma template. 🎯

   By considering these suggestions, you can make your portfolio project appear more polished, well-crafted, and in line with the desired design.

   ### Your design:
   ![image](https://github.com/santitoot/desktop/assets/107952724/bb06fbb0-7c17-4e64-8e96-6a4703101165)


   ### Figma design:
   ![image](https://github.com/santitoot/desktop/assets/107952724/0cbaa99a-fedc-4f9a-b6bd-d5988ef18048)